app/components/compensation_amount_component.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="sm:col-span-1">
+  <dt class="text-sm font-medium text-gray-500">
+    <%= title %>
+  </dt>
+  <dd class="flex mt-1 text-sm text-gray-900">
+    $<%= number_to_human(min_amount, format: "%n%u", units: {thousand: "K"}) %> -
```

Comment:
You can toss that `$` inside the `format` param.

```
number_to_human(min_amount, format: "$%n%u", units: {thousand: "K"})
```

Another approach is the following. Not sure which is more readable, though.

```
number_to_currency(min_amount/1000, precision: 0, format: "%u%nk")
```

Reply:
Fixed this, you can actually to `number_to_currency(number_to_human` :)